build-tools/packages/build-tools/src/common/gitRepo.ts

```diff
@@ -226,33 +226,58 @@ export class GitRepo {
 	}
 
 	/**
-	 * Returns an array containing repo repo-relative paths to all the files in the provided directory.
-	 * A given path will only be included once in the array; that is, there will be no duplicate paths.
-	 * Note that this function excludes files that are deleted locally whether the deletion is staged or not.
+	 * Returns git output of all non-ignored files including deleted files that are not staged.
+	 * A given path may be included multiple timed in the array if git version does not support
+	 * `--deduplicate` option (introduced in v2.31.0).
+	 * Note that this function excludes files that are deleted locally AND staged.
```

Comment:
This is different than what the other function returns?

Reply:
This returns the raw output rather than a list of files. This is helper for `getFiles`.